docs/user_manual/calculations.md

```diff
@@ -554,12 +563,104 @@ The assumptions used for calculations in power-grid-model are aligned to the one
+| [Automatic tap changing](#power-flow-with-automatic-tap-changing) | {py:meth}`tap_changing_strategy <power_grid_model.PowerGridModel.calculate_power_flow>` | {py:class}`TapChangingStrategy <power_grid_model.enum.TapChangingStrategy>` |
+
+#### Power flow with automatic tap changing
+
+```{warning}
+At the time of writing, this feature is still experimental and is not yet publicly available.
+```
+
+Some of the most important regulators in the grid affect the tap position of transformers.
+These {hoverxreftooltip}`user_manual/components:Transformer Tap Regulator`s try to regulate a control voltage $U_{\text{control}}$ such that it is within a specified voltage band.
+The $U_{\text{control}}$ may be compensated for the voltage drop during transport.
+Power flow calculations that take the behavior of these regulators into account may be toggled by providing one of the following strategies to the {py:meth}`tap_changing_strategy <power_grid_model.PowerGridModel.calculate_power_flow>` option.
+
+| Algorithm                                                              | Default  | Speed    | Algorithm call                                                                                              |
+| ---------------------------------------------------------------------- | -------- | -------- | ----------------------------------------------------------------------------------------------------------- |
+| No automatic tap changing (regular power flow)                         | &#10004; | &#10004; | {py:class}`TapChangingStrategy.disabled <power_grid_model.enum.TapChangingStrategy.disabled>`               |
+| Optimize tap positions for any value in the voltage band               |          | &#10004; | {py:class}`TapChangingStrategy.any_valid_tap <power_grid_model.enum.TapChangingStrategy.any_valid_tap>`     |
+| Optimize tap positions for lowest possible voltage in the voltage band |          |          | {py:class}`TapChangingStrategy.min_voltage_tap <power_grid_model.enum.TapChangingStrategy.min_voltage_tap>` |
+| Optimize tap positions for lowest possible voltage in the voltage band |          |          | {py:class}`TapChangingStrategy.max_voltage_tap <power_grid_model.enum.TapChangingStrategy.max_voltage_tap>` |
+
+##### Control logic for power flow with automatic tap changing
+
+The following control logic is used:
+
+- Regulated transformers are ranked according to how close they are to {hoverxreftooltip}`sources <user_manual/components:source>` in terms of the amount of transformers inbetween.
+  - Transformers are regulated in order according to their ranks.
+- Initialize all transformers to their starting tap position (see {hoverxreftooltip}`user_manual/calculations:Initialization and exploitation of regulated transformers`)
+- Find the optimal state using the following procedure
+  - While not all transformers are regulated, iterate as follows:
```

Comment:
This sounds like there is a state saying if a transformer is regulated already.
```suggestion
  - While not all transformers cannot be further regulated, iterate as follows:
```
or
```suggestion
  - While some transformers can still be further regulated, iterate as follows:
```